hazelcast/src/main/java/com/hazelcast/monitor/NearCacheStats.java

```diff
@@ -19,44 +19,58 @@
 public interface NearCacheStats extends LocalInstanceStats {
 
     /**
-     * Returns the creation time of this NearCache on this member
+     * Returns the creation time of this Near Cache on this member.
      *
-     * @return creation time of this NearCache on this member
+     * @return creation time of this Near Cache on this member.
      */
     long getCreationTime();
 
     /**
-     * Returns the number of entries owned by this member.
+     * Returns the number of Near Cache entries owned by this member.
```

Comment:
Does "entries owned by this member" make sense in the context of near cache? Perhaps we should amend the javadoc to "entries on this node" (applying to client as well)?